### PR TITLE
Rename files and create makefile

### PR DESCRIPTION
**Pull request by worksofliam**
Hi!

This PR includes the following changes:

* Remove `@` symbol from sources and program names. Wasn't sure what the need was.
* Implements a `makefile` to replace the `build.sh` script.
* Cleaned up readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,45 +4,52 @@ Welcome to IUNIT. The lightweight extensible unit testing framework for IBM i.
...
+
+* Do not depend one test on the result of another test  
+* A library is set of related tests. Create as many libraries as you need and call them sequentially  
+* When using [TD/OMS as your change management system.](https://remainsoftware.com) include tests in your compile or transfer   
 
 ## Tips
-Copy @SETUP and @TEARDOWN source from IUNIT into your test libraries and modify  
-You can copy IUNIT into your test library so that it is completely self contained  
+
+* Copy `SETUP` and `TEARDOWN` source from `IUNIT` into your test libraries and modify
```
Comment (wimjongman) [inline README.md:45]: Programs actually start with @ at sign.

File: README.md [hunk 2]
```diff
@@ -4,45 +4,52 @@ Welcome to IUNIT. The lightweight extensible unit testing framework for IBM i.
...
-Fail your test program by sending an escape message to *PRV with sndpgmmsg   
-Make sure the tests do not hang in QSYSOPR message queue by monitoring all commands. Otherwise they will stall your test  
-@SETUP is called before RUNONE and RUNALL run any tests  
-@TEARDOWN is called after RUNONE and RUNALL have ran all tests 
+
+* RUNALL runs all programs in the library that do not start with `@`
+* Fail your test program by sending an escape message to `*PRV` with sndpgmmsg   
+* Make sure the tests do not hang in `QSYSOPR` message queue by monitoring all commands. Otherwise they will stall your test  
+* `SETUP` is called before `RUNONE` and `RUNALL` run any tests  
+* `TEARDOWN` is called after `RUNONE` and `RUNALL` have ran all tests
```
Comment (wimjongman) [inline README.md:35]: @TEARDOWN

File: README.md [hunk 3]
```diff
@@ -4,45 +4,52 @@ Welcome to IUNIT. The lightweight extensible unit testing framework for IBM i.
...
-RUNALL runs all programs in the library that do not start with @  
-Fail your test program by sending an escape message to *PRV with sndpgmmsg   
-Make sure the tests do not hang in QSYSOPR message queue by monitoring all commands. Otherwise they will stall your test  
-@SETUP is called before RUNONE and RUNALL run any tests  
-@TEARDOWN is called after RUNONE and RUNALL have ran all tests 
+
+* RUNALL runs all programs in the library that do not start with `@`
+* Fail your test program by sending an escape message to `*PRV` with sndpgmmsg   
+* Make sure the tests do not hang in `QSYSOPR` message queue by monitoring all commands. Otherwise they will stall your test  
+* `SETUP` is called before `RUNONE` and `RUNALL` run any tests
```
Comment (wimjongman) [inline README.md:34]: @SETUP

**Comment by wimjongman** [comment]
>     * Remove `@` symbol from sources and program names. Wasn't sure what the need was.

All programs starting with @ will be ignored by the framework. This allows adding programs that are not tests but contain setup code and such.

**Comment by worksofliam** [comment]
@wimjongman The program names in the readme were updated alongside the renamed files in the initial commit.

**Comment by wimjongman** [comment]
@WorksOfLiam yeah, that is not possible. @ sign must stay.

All programs starting with @ will be ignored by the framework. This allows adding programs that are not tests but contain setup code and such.

**Comment by worksofliam** [comment]
@wimjongman 

This is the problem I am getting with the `@` symbol on IBM i with CCSID 277 using your `build.sh` script:

```
Fil QCMDSRC er oprettet i bibliotek IUNIT.
Stream file copied to object.
Member RUNALL file QCMDSRC in IUNIT changed.
Stream file copied to object.
Member RUNONE file QCMDSRC in IUNIT changed.
Characters in qualifier beginning '@BUILD) SR' not valid.
Strengen '@BUILD)   ' indeholder et ugyldigt tegn.
Errors occurred in command.
Characters in qualifier beginning '@BUILD PAR' not valid.
Errors occurred in command.
Done.
```

This is why I chose to rename them.

**Comment by worksofliam** [comment]
As a note, I also got the same issue with CCSID 273.

**Comment by wimjongman** [comment]
@WorksOfLiam Thanks but the names are functional. You cannot just rename them.  @ at the beginning of a name is valid, see the object name rules here: https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_72/rbam6/rbam6names.htm

The error you show here looks not related to the @ sign.

### Space and ) are not valid in a name
  Characters in qualifier beginning '@BUILD) SR' not valid.
  Strengen '@BUILD)   ' indeholder et ugyldigt tegn.

### Spaces are not valid in a name  
  Characters in qualifier beginning '@BUILD PAR' not valid.
 
I do wonder how a @ is handled across code pages since it is not part of the invariant set. Also a name can start with $ which is also not part of the invariant set.

Cheers

**Comment by worksofliam** [comment]
@wimjongman 

If this PR is of value, could you help fix the issue with building on non-37 systems? :) Otherwise, it may as well be closed for the time being.

**Comment by wimjongman** [comment]
Sure. Can you provide a login on such a server?

**Comment by wimjongman** [comment]
I filed issue #3 to fix this.

**Pull request closed without merging**